bot/exts/evergreen/duck_game.py

```diff
@@ -0,0 +1,329 @@
+            if old_footer == discord.Embed.Empty:
+                old_footer = ""
+            game_embed.set_footer(text=f"{old_footer}\n{str(answer):12s}  -  {author.display_name}")
+            await self.edit_embed_with_image(game.embed_msg, game_embed)
+
+    async def end_game(self, game: DuckGame, end_message: str) -> None:
+        """Edit the game embed to reflect the end of the game and mark the game as not running."""
+        game.running = False
+
+        scores = sorted(
+            game.scores.items(),
+            key=lambda item: item[1],
+            reverse=True,
+        )
+        scoreboard = "\n".join(f"{member.display_name}: {score}" for member, score in scores)
+
+        missed = [ans for ans in game.solutions if ans not in game.claimed_answers]
+        if missed:
+            missed_text = "Flights everyone missed:\n" + "\n".join(f"{ans}" for ans in missed)
+        else:
+            missed_text = "All the flights were found!"
+
+        game_embed, = game.embed_msg.embeds
+        old_footer = game_embed.footer.text
+        if old_footer == discord.Embed.Empty:
+            old_footer = ""
+
+        embed_as_dict = game_embed.to_dict()  # Cannot set embed color after initialization
+        embed_as_dict["color"] = discord.Color.red().value
+        game_embed = discord.Embed.from_dict(embed_as_dict)
+
+        game_embed.set_footer(
+            text=f"{old_footer.rstrip()}\n\n{end_message} Here are the scores:\n{scoreboard}\n\n{missed_text}"
+        )
+        await self.edit_embed_with_image(game.embed_msg, game_embed)
+
+    @start_game.command(name="help")
+    async def show_rules(self, ctx: commands.Context) -> None:
+        """Explain the rules of the game."""
+        await self.send_help_embed(ctx)
+
+    @start_game.command(name="stop")
+    async def stop_game(self, ctx: commands.Context) -> None:
+        """Stop a currently running game. Only available to mods."""
+        if not any(role in ctx.author.roles for role in MODERATION_ROLES):
+            return
+        try:
+            game = self.current_games.pop(ctx.channel.id)
+        except KeyError:
+            return
+        await self.end_game(game, end_message="Game canceled.")
+
+    @staticmethod
+    async def send_help_embed(ctx: commands.Context) -> discord.Message:
+        """Send rules embed."""
+        embed = discord.Embed(
+            title="Compete against other players to find valid flights!",
+            color=discord.Color.dark_purple(),
+        )
+        embed.description = (
+            "**Each card has 4 features**\nColor, Number, Hat, and Accessory\n"
+            "\n**A valid flight**\n3 cards where each feature is either all the same or all different\n"
+            '\n**Call "GOOSE"**\nif you think there are no more flights\n'
+            "\n**1**   for each valid flight\n"
+            '**2**   for a correct "GOOSE" call\n'
+            "**-1** for any wrong answer"
+        )
```

Comment:
I think this might be clearer as a multiline string constant defined at the top, then you can use actual newline characters instead of `\n`, making it more clear what the result will actually look like. Would also mean you don't have to use a mixture of single and double quoted strings.

Also not sure why there are 3 spaces, e.g. between `**1**` and  `for` on line 310, seems to render with just 1 in discord though, not sure if i'm missing something